csrc/ops.h

```diff
@@ -2,53 +2,41 @@
 
 #include <torch/extension.h>
 
-void paged_attention_v1(
-  torch::Tensor& out,
-  torch::Tensor& query,
-  torch::Tensor& key_cache,
-  torch::Tensor& value_cache,
-  int num_kv_heads,
-  float scale,
-  torch::Tensor& block_tables,
-  torch::Tensor& context_lens,
-  int block_size,
-  int max_context_len,
-  const c10::optional<torch::Tensor>& alibi_slopes);
+void paged_attention_v1(torch::Tensor &out, torch::Tensor &query,
```

Comment:
we should probably avoid reformatting this file, it'll cause headaches later on when syncing with main vLLM repo